src/Fixer/ClassNotation/SelfAccessorFixer.php

```diff
@@ -120,6 +120,11 @@ private function replaceNameOccurrences(Tokens $tokens, $name, $startIndex, $end
                 continue;
             }
 
+            // skip tokens that are part of class property
+            if ($prevToken->isGivenKind(T_OBJECT_OPERATOR)) {
```

Comment:
you can merge this check into the one above;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.2.4/src/Fixer/ClassNotation/SelfAccessorFixer.php#L119

```php
            // skip tokens that are part of a fully qualified name
            if ($prevToken->isGivenKind(T_NS_SEPARATOR) || $nextToken->isGivenKind(T_NS_SEPARATOR)) {
                continue;
            }
```

=>

```
            // skip tokens that are part of a fully qualified name or used in class property access
            if ($prevToken->isGivenKind([T_NS_SEPARATOR, T_OBJECT_OPERATOR]) || $nextToken->isGivenKind(T_NS_SEPARATOR)) {
                continue;
            }
```